### PR TITLE
Amend getting and providing feedback

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,4 +4,12 @@ We love pull requests. If you have something you want to add or remove, please o
 
 ## Getting Feedback
 
-Since these are our guides, we want everyone at DLSS to see them. We leave all PRs open for at least a week to get feedback from everyone.
+Since these are our guides, we want everyone at DLSS to have an opportunity to discuss them. To ensure a reasonable amount of time for discussion and consensus building, we try to leave all pull requests open for at least a week. 
+
+## Providing Feedback
+
+During discussion on a topic, participants are are welcome to offer feedback liberally to make sure all angles are considered. Objections should not be raised unless the individual strongly believes the decision is the wrong one in spite of discussion, and the individual cannot "live with" the decision. Compromise on points that the individual considers suboptimal but can "live with" is an essential part of group decisions that must meet various requirements. (Adapted from http://www.w3.org/WAI/PF/HTML/consensus-procedures)
```
Comment (cbeer) [inline CONTRIBUTING.md]: @azaroth42 as our resident W3C expert, is this a reasonable adaptation of the process? To my mind, the full process seemed too formal for our purposes.

**Comment by mejackreed** [comment]
:+1:

**Comment by ndushay** [comment]
maybe it's also "cannot live with" but can articulate the downside (and must articulate it for discussion)

**Comment by azaroth42** [comment]
:+1:

**Comment by darrenleeweber** [comment]
+1 but add additional information about how and why a PR is merged or not

**Comment by azaroth42** [comment]
More information should be added by further proposals please :)

**Comment by cbeer** [comment]
I think #21 supersedes this (although I plan to send a PR once it is merged to actually define what we mean by :+1: and :-1: using this language)

**Pull request closed without merging**